config.example.wikimedia.yaml

```diff
@@ -53,6 +53,20 @@ spec_root: &spec_root
     # global domain
     /{domain:wikimedia.org}: *wikimedia.org
 
+    # A robots.txt to make sure that the content isn't indexed.
```

Comment:
I'm hesitant to clutter the config any more, so I think x-amples is out. A separate test to assert that the robots.txt entry point is in the config doesn't seem to add a lot of value. The general handler functionality is tested elsewhere, so the only thing we'd assert is that the config has this configured in the test config, which says nothing about production.